test/setup/outputs.tf

```diff
@@ -0,0 +1,148 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "project_id" {
+  value = module.project.project_id
+}
+
+output "sub_folder_id" {
+  value = google_folder.ci_event_func_subfolder.id
+}
+
+output "sa_key" {
+  value     = google_service_account_key.int_test.private_key
+  sensitive = true
+}
+
+output "region" {
+  value = var.region
+}
+
+output "zone" {
+  value = var.zone
+}
+
+output "primary_zone" {
+  value = var.zone
+}
+
+output "secondary_zone" {
+  value = "us-east4-b"
+}
+
+output "subnetwork" {
+  value = module.network.subnets_self_links[0]
+}
+
+output "sap_hana_instance_number" {
+  value = 10
+}
+
+output "sap_hana_sid" {
```

Comment:
Instead of putting all these values as outputs here, it might be preferable to leave them as defaults in the fixture variables?

Reply:
I thought about that. But since I have 3 fixtures, often using the same vars, I thought it would be simpler to just define them once here.

Reply:
Got it, that seems reasonable.